src/madhavi_Raut/BankTransaction.java

```diff
@@ -0,0 +1,83 @@
+package madhavi_Raut;
+
+/*Assignment 09: 
+Create a Banking System which has the following functionality.
+
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit 
+
+& printBalance method called .
+Hint : Create 3 non static instance variable.
+Output :
+user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit 
+
+& printBalance method get called.
+Hint : Create 3 static variables will be required.
+Output :
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time*/
+public class BankTransaction {
+	double balance = 1000;
+	String customerName; 
+	int debitCountIndividual = 0, creditCountIndividual = 0, printCountIndividual = 0;
+	static int debitCountAll = 0, creditCountAll = 0, printCountAll = 0;
+	
+	void debitAmount(double debitAmt) {
+		balance -= debitAmt;
```

Comment:
You should check if (totalBalance >= requestedBalance ) then only you should debit amt from main balance